instrumentation/apolloconfig/apolloconfig-2.0.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apolloconfig/v2_0_0/ApolloRepositoryChangeInstrumentation.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.apolloconfig.v2_0_0;
+
+import static io.opentelemetry.javaagent.bootstrap.Java8BytecodeBridge.currentContext;
+import static io.opentelemetry.javaagent.instrumentation.apolloconfig.v2_0_0.ApolloConfigSingletons.REPOSITORY_CHANGE_REPEAT_CONTEXT_KEY;
+import static io.opentelemetry.javaagent.instrumentation.apolloconfig.v2_0_0.ApolloConfigSingletons.instrumenter;
+import static net.bytebuddy.matcher.ElementMatchers.named;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeTransformer;
+import net.bytebuddy.asm.Advice;
+import net.bytebuddy.description.type.TypeDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+
+public class ApolloRepositoryChangeInstrumentation implements TypeInstrumentation {
+
+  @Override
+  public ElementMatcher<TypeDescription> typeMatcher() {
+    return named("com.ctrip.framework.apollo.internals.AbstractConfigRepository");
+  }
+
+  @Override
+  public void transform(TypeTransformer transformer) {
+    String adviceName = this.getClass().getName() + "$ApolloRepositoryChangeAdvice";
```

Comment:
The variable looks like unnecessary here. Recommend to define here according to our convention, you can refer to other module like https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/4f946640418a305e96dcd1b726ce307f930cbccc/instrumentation/apache-httpclient/apache-httpclient-2.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachehttpclient/v2_0/ApacheHttpClientInstrumentation.java#L38-L45